section4/exercises/burrito.rb

```diff
@@ -11,9 +11,25 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+  def add_toppings(item)
+    @toppings << item
```

Comment:
Here is the kind of code I am talking about in my comments below- line 15 is what adds the item to your toppings array.